### ISSUE TITLE
CFXLua VS-Code ext. problem on MacOs

### ISSUE THREAD

**Issue by Jeremoiii** [overextended/cfxlua-vscode#30]
Hey, I've now had this issue for some time but I couldn't find any fix on my own. On MacOs some Natives aren't recognized by the lint and are just 'unkown globals' (as seen in the example image). I've already tried reinstalling the extention and vs-code but nothing seems to resolve this issue. 

On Windows everything runs fine so I'm unsure if it is because of me using this ext. on Mac but idk. 

Just ping me if you have any idea how to resolve this issue. Thanks in advance!

![image](https://github.com/overextended/cfxlua-vscode/assets/79488475/bd4c2b05-403d-4bb3-9798-debf598908c9)

---

## VS-Code Info:
Version: 1.86.2
Commit: 903b1e9d8990623e3d7da1df3d33db3e42d80eda
Date: 2024-02-13T19:42:12.210Z
Electron: 27.2.3
ElectronBuildId: 26908389
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Darwin arm64 23.3.0

## Extension:
v1.7.15
<img width="892" alt="Bildschirmfoto 2024-02-27 um 12 21 35" src="https://github.com/overextended/cfxlua-vscode/assets/79488475/716ba30d-68c2-42ea-8a2d-37be285eac54">

**Comment by BerkieBb** [comment]
> On Windows everything runs fine so I'm unsure if it is because of me using this ext. on Mac but idk.

I mean, that should speak for itself

**Comment by Jeremoiii** [comment]
Yeah, but it wasn't always the case. So I'm not sure since when and why it broke.

**Comment by Ehbw** [comment]
This is likely an issue with MacOS's version of vscode. Also FiveM/GTAV is not supported on MacOS anyway.
